bin/enable-executable

```diff
@@ -2,14 +2,18 @@
 
 require 'fileutils'
 
-puts 'Ensuring test files are executable...'
+puts 'Ensuring test files are executable.'
 
-(Dir.glob('**/*test.rb') + Dir.glob('bin/*')).each do |f|
-  if File.executable?(f)
+# Assume that this file lives in #{base}/bin
+base = File.join(__dir__,'..')
+files = Dir.glob("#{base}/**/*test.rb") + Dir.glob("#{base}/bin/*")
+
+files.each do |file|
+  if File.executable?(file)
     print '.'
   else
-    print "\nAdding exec bit to #{f}"
-    FileUtils.chmod('u+x', f)
+    print "\nAdding exec bit to #{file}"
+    FileUtils.chmod('u+x', file)
   end
```

Comment:
Replaced single letter variable `f` with the more descriptive `file`
It probably could be `filename`